src/admin.c

```diff
@@ -518,7 +522,11 @@ static bool admin_show_databases(PgSocket *admin, const char *arg)
 		if (db->host && strchr(db->host, ','))
 			load_balance_hosts_str = cf_get_lookup(&load_balance_hosts_lookup);
 
-		pktbuf_write_DataRow(buf, "ssissiiiissiiiiii",
+		target_session_attrs_lookup.value_p = &db->target_session_attrs;
+		if (db->target_session_attrs != TARGET_SESSION_ANY)
```

Comment:
Why keep the value NULL for `any`? (same for the other admin command below)

Reply:
This seems to be incorrect. Removed the conditional.